migrations/versions/7a6816e652f2_add_purchases_to_refund_requests.py

```diff
@@ -0,0 +1,30 @@
+"""Add purchases to refund requests
+
+Revision ID: 7a6816e652f2
+Revises: 214878041f05
+Create Date: 2024-04-20 22:41:56.453985
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '7a6816e652f2'
+down_revision = '214878041f05'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('purchase', sa.Column('refund_request_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(op.f('fk_purchase_refund_request_id_refund_request'), 'purchase', 'refund_request', ['refund_request_id'], ['id'])
+    op.add_column('purchase_version', sa.Column('refund_request_id', sa.Integer(), autoincrement=False, nullable=True))
```

Comment:
Sorta curious whether this is planned to have `autoincrement=True` in future? (basically I'm wondering whether the argument needs to be included)

Reply:
honestly not sure, this is the alembic/versioning code which I tend to leave alone unless I have reason to change

Reply:
Ah, got it.  Yep, that's fine then - thanks.